.github/workflows/test.yml

```diff
@@ -2,19 +2,22 @@ on: [push, pull_request]
 name: Test
 jobs:
   test:
+    strategy:
+      matrix:
+        go-version: [1.16.x, 1.18.x, 1.20.x]
```

Comment:
How about replacing `1.20.x` with `stable`?

```
go-version: [1.16.x, 1.18.x, stable]
```

Reply:
Cool. Didn't know about that.